### PR TITLE
fix #275 - cant run tests under windows

### LINKED ISSUE #275: Can't run unit tests on Windows
Running `test.sh` fails with an access violation.

Seems to be inside `std.allocator`'s unittests:

```
    dbghelp.dll!__SEH_prolog4_GS()  + 0x1a bytes    
    dbghelp.dll!LoadSymbols()  - 0x16 bytes 
    dbghelp.dll!_SymFunctionTableAccess64@12()  + 0x4a bytes    
    dbghelp.dll!FunctionTableAccessRoutineLocal()  + 0x13 bytes 
    dbghelp.dll!DbhStackServices::GetFunctionEntry()  + 0x20 bytes  
    dbghelp.dll!DbsX86StackUnwinder::Unwind()  + 0x70 bytes 
    dbghelp.dll!DbsStackUnwinder::DbhUnwind()  + 0xa0 bytes 
    dbghelp.dll!PickX86Walk()  + 0xc1 bytes 
    dbghelp.dll!_StackWalk64@36()  + 0xc8 bytes 
    test.exe!core@sys@windows@stacktrace@StackTrace@traceNoSync()  + 0x298 bytes    C++
    test.exe!core@sys@windows@stacktrace@StackTrace@__ctor()  + 0x2c bytes  C++
    ntdll.dll!ExecuteHandler2@20()  + 0x26 bytes    
    ntdll.dll!ExecuteHandler@20()  + 0x24 bytes 
    ntdll.dll!_RtlDispatchException@8()  + 0xd3 bytes   
    ntdll.dll!_KiUserExceptionDispatcher@8()  + 0xf bytes   
    test.exe!std@allocator@__unittestL3024_36()  Line 3024 + 0xe bytes  C++
>   test.exe!std@allocator@__modtest()  + 0x9e bytes    C++
    test.exe!core@runtime@runModuleUnitTests()  + 0x3f bytes    C++
    test.exe!object@ModuleInfo@opApply()  + 0xf bytes   C++
    test.exe!rt@minfo@moduleinfos_apply()  + 0x34 bytes C++
    test.exe!_runModuleUnitTests()  + 0x24 bytes    C++
    test.exe!rt@dmain2@_d_run_main()  + 0x11 bytes  C++
    test.exe!rt@dmain2@_d_run_main()  + 0x35 bytes  C++
    test.exe!__entrypoint@main(int argc=0x00000001, char * * argv=0x003e03c4)  Line 3 + 0x14 bytes  C++
    test.exe!_mainCRTStartup()  + 0xa9 bytes    C++
    kernel32.dll!@BaseThreadInitThunk@12()  + 0x12 bytes    
    ntdll.dll!___RtlUserThreadStart@8()  + 0x27 bytes   
    ntdll.dll!__RtlUserThreadStart@8()  + 0x1b bytes    
```

### PR DESCRIPTION
**Pull request by BBasile**
- delete the sh script: not used anymore and doesn't work under win.
- update the bat so that it can run the test using "test" as command line argument.

### CODE REVIEW

**Comment by rikkimax** [comment]
No errors when running ``build.bat test``.

**Comment by BBasile** [comment]
well you can approve then ;)

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ makefile has "ldc" and "gdc" targets if you'd prefer to compile with one of thes
 compilers instead of DMD. To install, simply place the generated binary (in the
 "bin" folder) somewhere on your $PATH.
 
+### Testing
+Testing doe not work with DUB.
```
Comment (BBasile) [inline README.md]: typo.

**Comment by BBasile** [comment]
No surprise, all green. @wilzbach, if one day you retry appveyor i inform you that the tests can be run under windows too.

**Pull request merged**